docs/application/tizen-studio/release-notes/4-0-release-notes.md

```diff
@@ -0,0 +1,60 @@
+# Tizen Studio 4.0 Release Notes
+
+- Release Date: Oct 31, 2020
+
+## IDE and Tools
+
+### New Features
+
+- IDE
+  - Tizen 6.0 platform support has been added.
+  - Battery Historian tool has been added to help analyze the battery usage.
+  - Tizen Core new CLI support for Tizen 6.0 Native applications has been added.
+  - Toochain has been updated to GCC 9.2 and LLVM 10.
+  - Support for macOS Catalina, Ubuntu 18.04, and Ubuntu 20.04 has been added.
+  - Support for aarch64 architecture on IoT 6.0 extension for RPi4 has been added.
+
+### Deprecated
+   - Tizen RT will be deprecated from Tizen Studio 4.1.
+   - Bridge Networking support in emulators will be deprecated in future Tizen Studio releases.
+  
+### Known Issues
+
+- Tizen Studio
+  - In macOS, if the UI perspectives are not displayed properly after updating Tizen Studio, it is recommended to restart the IDE. This issue is because of the uncleared cache from the previous installation.
+  - In case if you see any old perspective, it is recommended to create a new workspace and import the projects to the new workspace to resolve the issue.
+- Common
+  - If you install Tizen Studio in a directory that requires administrator privileges for access. For example, for **C:\ProgramFiles** location administrator privilege is needed to run the Tizen SDK tools. The Tizen Installer and the baseline SDK Installer alerts you if you try to install it into such a directory.
```

Comment:
@manishtr Although the current sentences give the meaning, but I am suggesting the following:  

```suggestion
  - If you install Tizen Studio in a directory such as **C:\ProgramFiles** that needs administrator privileges to run the Tizen SDK tools.  The Tizen Installer and the baseline SDK Installer alerts you.
```
If you like it, then incorporate, else just ignore.

Reply:
@nawab-samsung As we have to finish the review of this PR till today, and the last issue you reported is minor, 
we'd better to skip this comment.

Please remember this issue to check when release note for next Tizen Studio version is created.

If there is no other critical issue except this one, approve this PR.

Reply:
Sure, thanks